crates/ruff_linter/resources/test/fixtures/ruff/RUF027_0.py

```diff
@@ -68,3 +68,7 @@ def method_calls():
     first = "Wendy"
     last = "Appleseed"
     value.method("{first} {last}")  # RUF027
+
+def format_specifiers():
+    a = 4
+    b = "{a:b} {a:^5}"
```

Comment:
Does this test case panic on main? It's not panicking for me. I think we can use the same code as mentioned in the issue.

Reply:
Oh yeah, this wasn't meant to test for a panic, this was to test that we _do_ parse format specifiers correctly.

I'll add the code from the original issue as an additional snapshot test :)